aks/secrets/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "azure_resource_prefix" {
+  type        = string
+  description = "Prefix of Azure resources for the service"
+}
+
+variable "service_short" {
+  type        = string
+  description = "Short name of the service"
+}
+
+variable "config_short" {
+  type        = string
+  description = "Short name of the configuration"
+}
+
+variable "name" {
```

Comment:
Can we make this optional?
We may want to use this module for services with only 1 keyvault (almost all of them)

Reply:
Can do - I assumed we would migrate the services to use multiple key vaults as we migrated them to using these modules, since nothing is using them in production yet?

Reply:
I'd rather avoid making this mandatory to minimise the changes when migrating a service. Also, some services are simple and don't use infra secrets

Reply:
Okay, I've amended the commit to make the name optional!